includes/minify/class-html.php

```
@@ -82,6 +114,27 @@ public function __construct( $html, $options ) {
 			$this->exclude_delay_js = array_values( array_filter( $this->exclude_delay_js, 'strlen' ) );
 
+			// Cache delay JS strategy lists.
+			$this->delay_js_idle_list        = (array) Util::process_urls( $this->options['file_optimisation']['delayJSIdleList'] ?? array() );
+			$this->delay_js_viewport_list    = (array) Util::process_urls( $this->options['file_optimisation']['delayJSViewportList'] ?? array() );
+			$this->delay_js_default_strategy = ! empty( $this->options['file_optimisation']['delayJSDefaultStrategy'] )
+				? sanitize_text_field( $this->options['file_optimisation']['delayJSDefaultStrategy'] )
+				: 'interaction';
+
+			// Parse priority map.
+			$this->delay_js_priority = array();
+			$priority_raw            = Util::process_urls( $this->options['file_optimisation']['delayJSPriority'] ?? array() );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PerformanceOptimise\Inc\Util' in method '__construct'. (undefined)

(StaticAccess)